bodhi-server/bodhi-server.spec

```
@@ -35,6 +35,10 @@ BuildRequires:  createrepo_c
 BuildRequires:  skopeo
 BuildRequires:  dnf
 
+%if 0%{?fedora} && 0%{?fedora} >= 41
```

The first part of the conditional will always be true if the second part is.

```suggestion
%if 0%{?fedora} >= 41
```